test/e2e/resize_nodes.go

```diff
@@ -400,12 +439,30 @@ var _ = Describe("Nodes", func() {
 			waitForNodeToBe(c, node.Name, true, 2*time.Minute)
 
 			By("verify wheter new pods can be created on the re-attached node")
-			err = resizeReplicationController(c, ns, name, replicas+1)
-			Expect(err).NotTo(HaveOccurred())
-			err = waitForPodsCreatedRunningResponding(c, ns, name, replicas+1)
-			Expect(err).NotTo(HaveOccurred())
-			_, err = waitForRCPodOnNode(c, ns, name, node.Name)
-			Expect(err).NotTo(HaveOccurred())
+			additionalPod := "additionalpod"
+			expectNoError(createServeHostnamePodOnNode(c, ns, additionalPod, node.Name))
+			expectNoError(waitForPodsCreatedRunningResponding(c, ns, additionalPod, 1))
```

Comment:
Please use the Expect(err).NotTo(HaveOccurred()) pattern, or switch the entire file over.